database/migration/src/main/scala/cromwell/database/migration/failuremetadata/ExpandSingleFailureStrings.scala

```diff
@@ -0,0 +1,71 @@
+package cromwell.database.migration.failuremetadata
+
+import java.sql.{PreparedStatement, ResultSet}
+import cromwell.database.migration.custom.BatchedTaskChange
+
+/**
+  * Turns individual '| failures[0] | "blah" |' entries into '| failures[0]:message | "blah" |' and '| failures[0]:causedBy[] | NULL |'
+  */
+class ExpandSingleFailureStrings extends BatchedTaskChange {
+  val tableName = "METADATA_ENTRY"
+  val primaryKeyColumn = "METADATA_JOURNAL_ID"
+  val workflowIdColumn = "WORKFLOW_EXECUTION_UUID"
+  val metadataKeyColumn = "METADATA_KEY"
+  val callFqnColumn = "CALL_FQN"
+  val jobScatterIndexColumn = "JOB_SCATTER_INDEX"
+  val retryAttemptColumn = "JOB_RETRY_ATTEMPT"
+  val metadataTimestampColumn = "METADATA_TIMESTAMP"
+
+  val fixableFailureMessageFilter = "METADATA_KEY REGEXP '^failures\\\\[[0-9]*\\\\]$'"
+
+  override def readCountQuery: String = s"SELECT MAX($primaryKeyColumn) FROM $tableName;"
+
+  /**
+    * Find metadata keys in this range which can be fixed up.
+    */
+  override def readBatchQuery: String =
+    s"""|SELECT $primaryKeyColumn, $workflowIdColumn, $metadataKeyColumn, $callFqnColumn, $jobScatterIndexColumn, $retryAttemptColumn, $metadataTimestampColumn
+        | FROM $tableName
+        | WHERE $primaryKeyColumn >= ? AND $primaryKeyColumn < ?
+        | AND $fixableFailureMessageFilter
+        |""".stripMargin
```

Comment:
I'm following the pattern established in the other batch migrations. I guess there's no need for it to be a `def` though, I'll switch to a `val`